### DISCUSSION TITLE
Welcome to DarkflameServer Discussions!

### DISCUSSION THREAD

**Discussion by DarwinAnim8or** [DarkflameUniverse/DarkflameServer#1089] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MustacheMan2** [comment]
Hello. I'm MustacheMan12, but most call me MM12

I'm bound to this game whether I like it or not. I enjoy using the Happy Flower world editor, racing, and playing Battle of Nimbus Station. I also enjoy helping the best I can in documenting bugs so things can go smoothly. Real glad to have a proper discussion area, it's been long overdue. Happy coding!

**Comment by EmosewaMC** [comment]
hi im emosewamc, aka emo - literally a nerd doing funny stuff for the LEGO Universe community.  no clue how i got here but im happy with how far the server has come
